docs/xxjitserveraotcachepersistence.md

```diff
@@ -0,0 +1,64 @@
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:[+|-]JITServerAOTCachePersistence
+
+**(Linux&reg; only)**
+
+This option enables or disables the JITServer server's ability to allow its later instances to reuse AOT caches.
+
+## Syntax
+
+        -XX:[+|-]JITServerAOTCachePersistence
+
+| Setting                    | Effect  | Default                                                                              |
+|----------------------------|---------|:------------------------------------------------------------------------------------:|
+|`-XX:+JITServerAOTCachePersistence` | Enable  |                                                                                      |
+|`-XX:-JITServerAOTCachePersistence` | Disable | :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span> |
+
+## Explanation
+
+ With the `-XX:+JITServerAOTCachePersistence` option, the JITServer server periodically saves its AOT caches to files, allowing its other JITServer instances to load these caches from files the first time a client requests a particular named cache. It is useful to improve performance, especially in scenarios where JITServer instances are started up and shut down regularly.
+
+ This feature depends on the [`-XX:+JITServerUseAOTCache`](xxjitserveruseaotcache.md) command-line option, which is used to enable caching of AOT-compiled methods. You must specify this option both at the client JVM and at the server.
+
+ Use the [`-XX:JITServerAOTCacheName=<cache_name>`](xxjitserveraotcachename) option at the client to request for a particular AOT cache. If the requested cache does not exist in server's memory, server checks if the AOT cache persistence feature is enabled (`-XX:+JITServerAOTCachePersistence`), and whether a file for that cache exists. If the AOT cache file exists, server compiles the requested method, without waiting for the AOT cache to be loaded. If the AOT cache file does not exist or the AOT cache persistence feature is disabled (`-XX:-JITServerAOTCachePersistence`), the server creates an empty AOT cache and gradually populates it with AOT method bodies it compiles.
```

Comment:
"to request for" -> "to request"
"in server's memory, server" ->"at the server in-memory, the server"
"server compiles" -> "the server compiles"